core/src/main/java/org/bitcoinj/core/Transaction.java

```diff
@@ -761,10 +761,15 @@ public String toString(@Nullable AbstractBlockChain chain, @Nullable CharSequenc
         s.append('\n');
         if (updatedAt != null)
             s.append(indent).append("updated: ").append(Utils.dateTimeFormat(updatedAt)).append('\n');
-        if (version != MIN_STANDARD_VERSION)
-            s.append(indent).append("version ").append(version).append('\n');
-        Type type = (getVersionShort() == SPECIAL_VERSION) ? getType() : Type.TRANSACTION_NORMAL;
-        s.append("  type ").append(type.toString()).append('(').append(type.getValue()).append(")\n");
+        if (version != MIN_STANDARD_VERSION) {
+            if (getVersionShort() == SPECIAL_VERSION) {
+                s.append(indent).append("version: ").append(getVersionShort()).append('\n');
+                Type type = (getVersionShort() == SPECIAL_VERSION) ? getType() : Type.TRANSACTION_NORMAL;
+                s.append("   type: ").append(type.toString()).append('(').append(type.getValue()).append(")\n");
+            } else {
+                s.append(indent).append("version: ").append(version).append('\n');
+            }
+        }
```

Comment:
This is to make the version number nicer to look at.  This should also be added to master.